frontend/public/style/_common.scss

```diff
@@ -219,6 +219,10 @@ input[type=number]::-webkit-outer-spin-button {
   display: inline-block;
   padding-right: 25px;
 }
+// Enable break word within co-m-table-grid
+.co-external-link--block-wrap {
```

Comment:
I might call this `.co-external-link--block` instead since the wrapping is a side effect?